user_docs/en/userGuide.t2t

```diff
@@ -2361,19 +2361,24 @@ With several historically popular GUI APIs, the text may be rendered with a tran
 : Default
   Disabled
 : Options
-  Disabled, Enabled
+  Default (Disabled), Enabled, Disabled
 :
 
 This option enables audio output via the Windows Audio Session API (WASAPI).
 WASAPI is a more modern audio framework which may improve the responsiveness, performance and stability of NVDA audio output, including both speech and sounds.
 After changing this option, you will need to restart NVDA for the change to take effect.
 
 ==== Volume of NVDA sounds follows voice volume ====[SoundVolumeFollowsVoice]
+: Default

```

Comment:
So do you confirm that this new presentation of options is not only for feature floags / combo-box but also for checkboxes?

Reply:
I think it makes sense, as a way to indicate the default.
I've updated the user guide standards doc to be explicit.